src/example_package/csv_reader/csv_reader.py

```diff
@@ -0,0 +1,52 @@
+from tkinter import filedialog, messagebox
+from config import constraints
+
+import pandas as pd
+
+df = None
+
+
+def validateCSV(df):
+
+    # Check if all required columns are present
+    if not all(col in df.columns for col in constraints['required_columns']):
+        raise ValueError(f"Columns {constraints['required_columns']} are missing in the CSV file.")
+
+    for index, row in df.iterrows():
+
+        # Check if age category is valid
+        if row['age_category'] not in constraints['age_categories']:
+            raise ValueError(f"Invalid age category {row['age_category']} at row {index + 1}")
+
+        # Check if weight category is valid
+        if row['weight_category'] not in constraints['age_categories'][row['age_category']]:
+            raise ValueError(f"Invalid weight category {row['weight_category']} at row {index + 1}")
+
+        # Check if weight is within the valid range
+        weight = row['weight']
+        min_weight = constraints['age_categories'][row['age_category']][row['weight_category']]['min']
+        max_weight = constraints['age_categories'][row['age_category']][row['weight_category']]['max']
+        if weight < min_weight or weight > max_weight:
```

Comment:
Można w sumie skorzystać z pythonowych udogodnień składni.
```suggestion
        if not min_weight <= weight <= max_weight:
```
Tylko trzeba jeszcze pomyśleć, czy zrobić nierówności ostre czy nieostre. Moja sugestia powinna być równoważna obecnemu kodowi.